documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -2466,6 +2468,14 @@ The following table lists relevant properties for configuring parallel execution
 | a positive decimal number
 | ```1.0```
 
+| ```junit.jupiter.execution.parallel.config.dynamic.max-pool-size.factor```
```

Comment:
Why are we using three backticks rather than one in this table?

Reply:
I believe there was a formatting error without the triple backticks.

But maybe that's no longer an issue with more recent versions of Asciidoctor.

So feel free to see what happens if you switch to single backticks.

Reply:
> I believe there was a formatting error without the triple backticks.

This is still the case.